elfi/model/extensions.py

```diff
@@ -16,6 +23,7 @@ def __init__(self, name=None):
         """
         self._name = name or self.__class__.__name__
 
+    @classmethod
```

Comment:
So this is not meant to be used by others implementing their custom distributions?